haystack/modeling/model/adaptive_model.py

```diff
@@ -626,8 +626,8 @@ def convert_to_onnx(
                                https://huggingface.co/transformers/main_classes/model.html#transformers.PreTrainedModel.from_pretrained
         :return: None.
         """
-        language_model_class = LanguageModel.get_language_model_class(model_name)
-        if language_model_class not in ["Bert", "Roberta", "XLMRoberta"]:
+        language_model = get_language_model(model_name)
```

Comment:
You might use here this function: https://github.com/deepset-ai/haystack/blob/8fbccbda8232648be409cdcd66a909bc99928920/haystack/modeling/model/language_model.py#L939

Reply:
Will try, thanks Bogdan